event-sourcing/subscription-manager/src/main/java/uk/gov/justice/services/event/sourcing/subscription/manager/CatchupEventBufferProcessor.java

```diff
@@ -27,12 +27,12 @@
     private InterceptorChainProcessorProducer interceptorChainProcessorProducer;
```

Comment:
I think this class (CatchupEventBufferProcessor) is missing a test

Reply:
CatchupEventBufferProcessorTest added